api/rooms/views/main.py

```diff
@@ -0,0 +1,22 @@
+import requests, json
+
+from django.views     import View
+from django.http      import JsonResponse
+
+from my_settings import API_KEY
+
+class SearchRoomView(View):
+    def get(self, request):
+        try:
+            search_address = request.GET.get('search_address', None)
+            search_address = search_address.strip()
+            type           = request.GET.get('type')
+            category       = request.GET.get('category')
+           
+            if len(search_address) < 2 : 
+                    return JsonResponse({'message':'ADDRESS_IS_TOO_SHORT'}, status=400)
+            response= requests.get(f'http://api.vworld.kr/req/search?service=search&request=search&version=2.0&crs=EPSG:900913&size=100&page=1&query={search_address}&type={type}&category={category}&format=json&errorformat=json&key={API_KEY}')
+            search_data = response.json()
```

Comment:
반환되는 데이터는 어떤 형태인가요? 예시를 코멘트에 달아주실 수 있나요?

```suggestion
            search_data = response.json()
```

Reply:
넵